### ISSUE TITLE
[BUG]: v5.5 crashing

### ISSUE THREAD

**Issue by ramierop** [PhenoApps/Field-Book#706]
### Description

Trying to use v5.5 on my Samsung S22. When opening Field Import, long lag time even with only sample fields and my intended field loaded followed by crashing. I get the attached message. 
![-3480267993938686699](https://github.com/PhenoApps/Field-Book/assets/70235332/6a50db7d-8503-496b-8324-289f2d3fd211)

### Field Book version

5.5

**Comment by ramierop** [comment]
I will add that I am importing from Local Storage

**Comment by trife** [comment]
@chaneylc 

````Fatal Exception: java.lang.NoClassDefFoundError: jxl.WorkbookSettings
       at jxl.WorkbookSettings.class$(WorkbookSettings.java:12)
       at jxl.WorkbookSettings.<clinit>(WorkbookSettings.java:6)
       at com.fieldbook.tracker.objects.FieldFileObject$FieldFileExcel.getColumns(FieldFileObject.java:4)
       at com.fieldbook.tracker.activities.FieldEditorActivity.loadFile(FieldEditorActivity.java)
       at com.fieldbook.tracker.activities.FieldEditorActivity.showFieldFileDialog(FieldEditorActivity.java:144)
       at com.fieldbook.tracker.activities.FieldEditorActivity.onActivityResult(FieldEditorActivity.java:30)
       at android.app.Activity.dispatchActivityResult(Activity.java:8951)
       at android.app.ActivityThread.deliverResults(ActivityThread.java:5987)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:6033)
       at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:67)
       at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2574)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loopOnce(Looper.java:226)
       at android.os.Looper.loop(Looper.java:313)
       at android.app.ActivityThread.main(ActivityThread.java:8757)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)````

@ramierop can you post or send the file that's causing the issue?

**Comment by ramierop** [comment]
Yes, I will load that this afternoon

**Comment by ramierop** [comment]
[23StorageRating.xls](https://github.com/PhenoApps/Field-Book/files/12294946/23StorageRating.xls)

Attached is the file. Let me know if you need anything else.

**Comment by trife** [comment]
@ramierop I think I know what the issue is but won't be able to confirm until I'm back in the office on Thursday. Until then, you should be able to save this as a csv and import without issue.

@chaneylc I think this is more fallout from minify

**Comment by chaneylc** [comment]
@trife easiest solution for now is to disable minify on release, until I figure out a better solution

**Comment by trife** [comment]
fixed in #709
